Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -30,20 +30,24 @@ struct DashboardView: View {
         NavigationStack {
             List {
                 ForEach(viewModel.searchedEvents) { event in
-                    VStack(alignment: .leading, spacing: 8) {
+                    VStack(alignment: .leading) {
                         Text(event.title)
                             .font(.title3)
+                            .fontWeight(.bold)
+                        
+                        Text(event.date, formatter: self.eventDateFormat)
+                            .foregroundStyle(.secondary)
                         
                         let vehicleName = viewModel.vehicles.first { $0.id == event.vehicleID }?.name
                         if let vehicleName {
-                            Text("\(vehicleName) on \(event.date, formatter: self.eventDateFormat)",
-                                 comment: "Maintenance list item for a vehicle on a date")
+                            Text("For: \(vehicleName)")
```

Comment:
adding a comment here for accessibility when people are translating this it gives more context about that text